pallets/weights/Cargo.toml

```diff
@@ -39,7 +39,7 @@ pallet-sto = { path = "../sto", default-features = false  }
 pallet-treasury = { path = "../treasury", default-features = false }
 pallet-utility = { path = "../utility", default-features = false }
 pallet-test-utils = { path = "../test-utils", default-features = false }
-#polymesh-contracts = { path = "../contracts", default-features = false }
+polymesh-contracts = { path = "../contracts2", default-features = false }
```

Comment:
We already have an old an unused `contracts` pallet, so I wanted to avoid renaming it here. I've moved it to `contracts-old` now and this one to `contracts`.